packages/iTwinUI-react/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "configs/tsconfig.base.json",
```

Comment:
is this path correct? it shows error to me

Reply:
It is correct. `configs` is the name of the package where I've moved this file.

https://user-images.githubusercontent.com/9084735/164474724-79c0a9fd-5dd4-42f5-ae27-68e16c623fff.mov